openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -259,3 +242,34 @@ def get_credit_provider_display_names(course_key):
         cache.set(cache_key, provider_names, credit_config.cache_ttl)
 
     return provider_names
+
+
+def make_providers_strings(providers):
+    """Get the list of course providers and make them comma seperated string.
+
+    Arguments:
+        providers : List containing the providers names
+
+    Returns:
+        strings containing providers names in readable way .
+    """
+    providers_string = None
+    if providers:
+        if len(providers) > 1:
+            if len(providers) > 2:
+                # Translators: The join of three or more university names. The first of these formatting strings
+                # represents a comma-separated list of names (e.g., MIT, Harvard, Dartmouth).
+                providers_string = _("{first_providers}, and {last_provider}").format(
+                    first_providers=u", ".join(providers[:-1]),
+                    last_provider=providers[-1]
+                )
+            else:
+                # Translators: The join of two university names (e.g., Harvard and MIT).
+                providers_string = _("{first_provider} and {second_provider}").format(
+                    first_provider=providers[0],
+                    second_provider=providers[1]
+                )
+        else:
+            providers_string = providers[0]
+
```

Comment:
This else is entered if `len(providers) == 1`. I'd suggest switching to `if len(providers) == 1: providers_string = providers[0]`.